snapcraft/cli/remote.py

```diff
@@ -15,11 +15,9 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 import click
-import os
-import uuid
 
 from snapcraft.project import Project
-from snapcraft.internal.remote_build import WorkTree, LaunchpadClient, InfoFile, errors
```

Comment:
you didn't remove InfoFile in this commit(?)

Reply:
I purposely left that in until the config bits are sorted out (maybe it could be re-used?).  Just thought I'd punt to then anyhow so I didn't have to come to a conclusion on that just yet :)